Makefile

```diff
@@ -113,7 +114,7 @@ release-checks: check-venv
 	. $(VENV_ACTIVATE_FILE); ./release-checks.sh $(release_version) $(next_version)
 
 # usage: e.g. make release release_version=0.9.2 next_version=0.9.3
-release: check-venv release-checks clean docs it
+release: check-venv release-checks clean docs test it
```

Comment:
Is the intention to run the entirety of tests prior to `release`? If yes, should we include `lint` in this target too?

Reply:
```suggestion
release: check-venv release-checks clean docs lint test it
```